### ISSUE TITLE
Export not working for excel nor yaml, but for json

### ISSUE THREAD

**Issue by psikonetik** [Benny-/Yahoo-ticker-symbol-downloader#48]
Hi,

I keep receiving this message when doing
`python YahooTickerDownloader.py -e`
`Exporting generic symbols`
`Could not export .xlsx due to a internal error`

Do you know what could be causing this issue?

thanks

**Comment by Benny-** [comment]
Nope, I don't. Can you send me the .pickle file?

**Comment by psikonetik** [comment]
oh hold on did some debugging. It exports nice with python 2.7 but not with 3.7
Do you still need the pickle?
Thanks master!

**Comment by Benny-** [comment]
Yes, please. Sending me the pickle file will allow me to debug why it doesn't work in 3.7 and fix it.

This script should support python 2 as well as python 3.

**Comment by AmericanEagle1989** [comment]
I am having the same issue. I have Python 3.9.0

"The downloader has already finished downloading everything

Exporting generic symbols
Could not export .xlsx due to a internal error
Could not export .yaml due to a internal error

Process finished with exit code 0"

Have you found a solution?

**Comment by AmericanEagle1989** [comment]
Actually, it's all good. Even though it gives that message, I checked if the file wasn't by any chance in the folder and it is so I'm good. :-)
